rpc/core/mempool.go

```diff
@@ -59,6 +59,7 @@ func BroadcastTxSync(ctx *rpctypes.Context, tx types.Tx) (*ctypes.ResultBroadcas
 	}
 }
 
+// DEPRECATED: Use BroadcastTxSync or BroadcastTxAsync alongside TxStatus instead.
```

Comment:
[question] what do you mean by alongside `TxStatus`? I would expect users to invoke `BroadcastTxSync` or `BroadcastTxAsync` and then subsequently invoke `TxStatus`. Maybe we can remove `alongside TxStatus` here and add to the Go doc of `BroadcastTxSync` and `BroadcastTxAsync` that users can invoke `TxStatus` after.

```suggestion
// DEPRECATED: Use BroadcastTxSync or BroadcastTxAsync instead.
```